bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/AngularVelocity.scala

```diff
@@ -0,0 +1,63 @@
+package edu.gemini.spModel.core
+
+import scalaz._, Scalaz._
+
+/** An angular velocity in [mas/y]  */
+case class AngularVelocity(masPerYear: Double) extends AnyVal with java.io.Serializable {
+
+  /**
+   * This `AngularVelocity` in degrees/year, it can be positive or negative
+   * @group Conversions
+   */
+  def toDegreesPerYear: Double = masPerYear / AngularVelocity.MilliArcSecsInADegree
+
+}
+
+/** An Angular Velocity in Right Ascension */
+case class RightAscensionAngularVelocity(velocity: AngularVelocity)
+
+/** An Angular Velocity in Declination */
+case class DeclinationAngularVelocity(velocity: AngularVelocity)
+
+object AngularVelocity {
+  val MilliArcSecsInADegree:Double = 1296000000
+
+  val Zero = AngularVelocity(0.0)
+
+  /** @group Typeclass Instances */
+  implicit val order: Order[AngularVelocity] =
+    Order.orderBy(_.masPerYear)
+
+  /** @group Typeclass Instances */
+  implicit val ordering: scala.Ordering[AngularVelocity] =
+    scala.Ordering.by(_.masPerYear)
+
+  /** @group Typeclass Instances */
+  implicit val equal: Equal[AngularVelocity] =
+    Equal.equalA[AngularVelocity]
+  
+  /** @group Typeclass Instances */
+  implicit val monoid: Monoid[AngularVelocity] =
+    Monoid.instance((a, b) => AngularVelocity(a.masPerYear + b.masPerYear), Zero)
```

Comment:
I can imagine you could make a generic function to print and change the format by providing a different instance of `Show`, e.g. one that could restrict output to 80 chars